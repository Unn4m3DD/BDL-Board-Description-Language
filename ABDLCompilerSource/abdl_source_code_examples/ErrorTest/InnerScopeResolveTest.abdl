main:
    if(0) then
        let i = 0;
    end if;
    i = 0;
end main;