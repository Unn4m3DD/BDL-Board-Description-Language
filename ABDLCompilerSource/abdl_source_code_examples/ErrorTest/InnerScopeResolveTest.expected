Variable not declared (5:4): i